build.gradle

```diff
@@ -84,6 +84,7 @@ subprojects { subproject ->
 		log4jVersion = '1.2.17'
 		mockitoVersion = '1.9.5'
 		scalaVersion = '2.11'
+		slf4jVersionRuntime = '1.7.21'
```

Comment:
It doesn't matter how we will use the variable. Our code convention here is just `...Version`
Will fix on merge.